tests/integration-tests/tests/arm_pl/test_arm_pl/test_arm_pl/pcluster.config.yaml

```diff
@@ -12,7 +12,7 @@ Scheduling:
     - Name: queue-0
       ComputeResources:
         - Name: compute-resource-0
-          {% if scheduler == "Awsbatch" %}
+          {% if scheduler == "awsbatch" %}
```

Comment:
are we sure the string is "awsbatch" here? Cause I also see that the same is assigned to Scheduler

Reply:
yes the only place on which we're using the string "awsbatch" is as a value for the Scheduler parameter.